lib/config-initializer.js

```diff
@@ -26,6 +26,22 @@ function writeFile(config, isJson, callback) {
         callback(e);
         return;
     }
+
+    // install any external configs as well as any included plugins
+    if (config.extends && config.extends.indexOf("eslint") === -1) {
```

Comment:
Extends can be an array, and they might have included `eslint-config-` already, so you don't always want to add it.